include/imu_ros2/imu_diag_ros_publisher.h

```diff
@@ -27,6 +27,10 @@
 #include <rclcpp/rclcpp.hpp>
 #include <std_msgs/msg/string.hpp>
 
+#define USER_PARAM_SETTING_MODE 0
+#define DEVICE_PARAM_SETTING_MODE 1
+#define DEVICE_CONTINUOUS_SAMPLING_MODE 2
+
```

Comment:
you can avoid the redefinition of these values by adding them in a header file and include the header file where needed